### PR TITLE
Test multichain statistics endpoint

### PR DESCRIPTION
**Pull request by yinghaodai**

### CODE REVIEW

File: Tribler/Test/Core/Modules/RestApi/test_multichain_statistics_endpoint.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
+import json
+from twisted.internet.defer import inlineCallbacks
```
Comment (vandenheuvel) [inline Tribler/Test/Core/Modules/RestApi/test_multichain_statistics_endpoint.py]: Please add a newline between these two imports.

File: Tribler/Test/Core/Modules/RestApi/test_multichain_statistics_endpoint.py [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    @deferred(timeout=10)
+    def test_get_statistics_no_community(self):
+        """
+        Testing whether the API returns error 404 if no multichain community is loaded
+        """
+        dummy_master_member = DummyMember(self.dispersy, 1, "b" * 20)
+        Community.__abstractmethods__ = frozenset()
+        self.dispersy.get_communities = lambda: [Community(self.dispersy, dummy_master_member, self.member),
+                                                 Community(self.dispersy, dummy_master_member, self.member)]
+        return self.do_request('/display?focus_node=30&neighbor_level=1', expected_code=404)
```
Comment (vandenheuvel) [inline Tribler/Test/Core/Modules/RestApi/test_multichain_statistics_endpoint.py]: Please remove this `/` at the start of the path.

File: Tribler/Test/Core/Modules/RestApi/test_multichain_statistics_endpoint.py [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+            for edge in list_of_edges:
+                self.assertIn("from", edge)
+                self.assertIn("to", edge)
+                self.assertIn("amount", edge)
+
+        self.should_check_equality = False
+        request = 'display?focus_node=' + public_key + '&neighbor_level=' + str(neighbor_level)
+        return self.do_request(request,
+                               expected_code=200).addCallback(verify_response)
+
```
Comment (vandenheuvel) [inline Tribler/Test/Core/Modules/RestApi/test_multichain_statistics_endpoint.py]: Please remove this final newline.

File: Tribler/Test/Core/Modules/RestApi/test_multichain_statistics_endpoint.py (hunk 2 above)
Comment (yinghaodai) [inline Tribler/Test/Core/Modules/RestApi/test_multichain_statistics_endpoint.py]: Wow what an attention for detail! Will fix it :)

**Comment by MaxVanDeursen** [comment]
retest this please

**Comment by MaxVanDeursen** [comment]
Note about the failed checks:

In order to make the GUI tests work, the QtWebEngineWidgets has to be imported here because otherwise an ImportError will be thrown. This is troubleshooted extensively in this StackOverflow post and seems to be necessary.
Since the Linux and Mac devices do not yet have this dependency, which means that this is a TODO for the future.

The PyLint error is because of accessing a single method from a protected attribute which cannot be called in another way. Since this is the usual way to be done in Tribler, this will be done in this PR as well.

**Pull request merged**